docs/vision.md

```diff
@@ -72,12 +72,65 @@ moved to GraphQL and (the yet young) Apollo client to embrace their rich and
+It is composed of a React application, a GraphQL middleware using Magento2
+Web APIs, Redis for caching and ElasticSearch for search — more details on
+[Front-Commerce’s architecture documentation page](architecture-overview.md).
 
-## What do we want it to be
+We wanted to be our first integrators (dogfooding), and to create a few stores with
+Front-Commerce before distributing the product to the world.
+
+Being in production means that we focused on features, themes, payments,
+deployments, caching, logging… and everything you would expect from such a
+solution. **That makes us the most mature « eCommerce PWA theme » in our field.**
+
+We are now working on technical documentation and refactoring (v1.0), along with
+business and legal things to start selling license to customers and help
+them building their stores with Front-Commerce. It also means that we are
+building an ecosystem of partners. See [the license page](/license.html)
+for further information.
 
-Proin eget tortor risus. Praesent sapien massa, convallis a pellentesque nec, egestas non nisi. Cras ultricies ligula sed magna dictum porta.
+**The product is alive, and ready for you to use!** We are now entering in a
+release cycle to [deliver incremental improvements regularly](roadmap.md).
+
+## What do we want it to be
 
-Pellentesque in ipsum id orci porta dapibus. Proin eget tortor risus. Curabitur aliquet quam id dui posuere blandit.
+Over the years, it became clear that Front-Commerce (and PWA in general) was
+focused on users and their shopping experience.
+
+In a web environment it means that we have to match high quality standards.
+Hence our vision is to **sell a product that contains all the best practices
+for building qualitative online stores by default** so that teams can then
+focus _only_ on what is relevant to their context.
+
+Best practices means:
+
+* efficient collaboration between people and teams
+* web performance
+* accessibility
+* SEO
+* consistent web UIs
+* new web APIs
+* different types of tests (unit, integration, visual regression, contracts…)
+* continuous deployment
+* microservices architecture and resilient websites
+
+Our field evolves at a fast pace and **it is difficult to remain aware of all
+the best practices**. By using Front-Commerce, you will benefit from all our
+experience and get those new fancy practices with our regular updates:
+**your stores will perform better over time**.
+
+Relieved from this burden, we hope that **you will focus more on your
+users** and deliver the shopping experience they deserve and love.
+
+As developers, you will also learn new concepts and have everything you
+need to experiment what you never took the time to learn before.
+**If you are working on a team and want to increase the overall quality,
+Front-Commerce can help you introduce better practices**.
+
+We believe that together we can build a faster and more usable web,
+starting with online stores. Are you ready for this revolution?
```

Comment:
Ideally we could add a call to action here. To the getting started/ receipe page ? Or the roadmap since the other pages are not ready yet.